<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>119</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>36</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 36pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>准备!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="title">
     <string>操作</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>串口设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>26</y>
        <width>101</width>
        <height>188</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>端口:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="COMBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>数据位:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="DataBox">
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>停止位:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="StopBox">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>26</y>
        <width>101</width>
        <height>124</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>波特率:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="SpeedBox">
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>校验位:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CheckBox">
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>奇效验</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>偶效验</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ChuanKouButton">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>160</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>比赛</string>
     </property>
     <widget class="QPushButton" name="ConfirmButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提交成绩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ChuJieButton">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>31</y>
        <width>141</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车模出界</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StopFailButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>停车失败</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ChujieLine">
      <property name="geometry">
       <rect>
        <x>163</x>
        <y>35</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>31</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>次</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>239</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>66</y>
        <width>221</width>
        <height>117</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>添加队伍</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PowerButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>比赛开始</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="AboutButton">
           <property name="text">
            <string>关于此软件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AboutQtButton">
           <property name="text">
            <string>关于Qt</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>1151</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="title">
     <string>队伍</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1131</width>
       <height>601</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>20</y>
      <width>631</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 25 36pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>队名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NameLine">
       <property name="styleSheet">
        <string notr="true">font: 25 30pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>23</y>
      <width>215</width>
      <height>124</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 72pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLine">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>25</y>
      <width>258</width>
      <height>124</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 72pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>00:00:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLine_MS">
    <property name="geometry">
     <rect>
      <x>625</x>
      <y>25</y>
      <width>161</width>
      <height>124</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 72pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>802</x>
      <y>92</y>
      <width>108</width>
      <height>62</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 36pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>组别:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TypeBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>90</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>光电四轮组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>光电直立组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电磁普通组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电磁节能组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>光电追逐组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电磁追逐组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>双车对抗组</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四旋翼导航组</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>90</y>
      <width>108</width>
      <height>62</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 36pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string>序号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NumberLine">
    <property name="geometry">
     <rect>
      <x>1359</x>
      <y>93</y>
      <width>71</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;微软雅黑 Light&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
